### ISSUE TITLE
Add "Play Store" item on longpress menu in Sesame Search

### ISSUE THREAD

**Issue by rodrigoswz** [BranchMetrics/sesame_issues#87]
There are several items like info, uninstall, add to homescreen, edit shortcuts... But I also wanted a button to visit this app in the Play Store.

**Comment by philwall2** [comment]
Good idea. We've gotten this request before. I'll leave this up here and mark it a Medium priority. If other ppl thumbs up or comment we'll bump it to High.
